flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/operations/SqlNodeToOperationConversion.java

```diff
@@ -920,6 +941,15 @@ private Operation convertRichExplain(SqlRichExplain sqlExplain) {
         return new ExplainOperation(operation, sqlExplain.getExplainDetails());
     }
 
+    /** Convert DESCRIBE MODEL [EXTENDED] [[catalogName.] dataBasesName].sqlIdentifier. */
+    private Operation convertDescribeModel(SqlRichDescribeModel sqlRichDescribeModel) {
```

Comment:
```suggestion
    /** Convert DESCRIBE MODEL [EXTENDED] [[catalogName.] dataBasesName.]sqlIdentifier. */
    private Operation convertDescribeModel(SqlRichDescribeModel sqlRichDescribeModel) {
```
seems similar issue for the javadoc for the method below